### ISSUE TITLE
4.2.0 breaks siteRoot configuration variable

### ISSUE THREAD

**Issue by TomHAnderson** [magidoc-org/magidoc#337]
### Describe the bug

Under all circumstances when a website.options.siteRoot is specified in the config magidoc.mjs, the build will break.

```
Command 'pnpm' failed with status 1 when executed in directory /var/folders/0w/4w2_lc2j6rg0j39hd2sycd9w0000gp/T/carbon-multi-page@4.2.0
```

```
node:internal/event_target:1096
  process.nextTick(() => { throw err; });
                           ^
Error: 404 /types/Float does not begin with `base`, which is configured in `paths.base` and can be imported from `$app/paths` - see https://kit.svelte.dev/docs/configuration#paths for more info (linked from /docs/queries/event)
To suppress or handle this error, implement `handleHttpError` in https://kit.svelte.dev/docs/configuration#prerender
```

Downgrading to 4.1.4 fixes this problem.

### Reproduction

Create a build in 4.2.0 with a siteRoot specified

### Logs

```shell
[Pro@~/Projects/apiskeletons/stayker/livenation-admin (feature/improved-docs)]$ pnpm add --global @magidoc/cli@4.2.0 && magidoc generate
Packages: +12 -11
++++++++++++-----------
Progress: resolved 87, reused 87, downloaded 0, added 0, done

/Users/tom.h.anderson/Library/pnpm/global/5:
- @magidoc/cli 4.1.4
+ @magidoc/cli 4.2.0

Done in 1.4s
✔ Determine tmp directories [0.0s]
✔ Select Package Manager [0.0s]
↓ Template already unzipped.
↓ Template already unzipped
✔ Install dependencies [0.4s]
✔ Resolving template configuration [0.0s]
✔ Load GraphQL Schema [1s]
✔ Write variables file [0.0s]
✖ Build template
◼ Move output
Command 'pnpm' failed with status 1 when executed in directory /var/folders/0w/4w2_lc2j6rg0j39hd2sycd9w0000gp/T/carbon-multi-page@4.2.0

---- Program Output----

> @magidoc/starter-carbon-multi-page@4.2.0 build /private/var/folders/0w/4w2_lc2j6rg0j39hd2sycd9w0000gp/T/carbon-multi-page@4.2.0
> svelte-kit sync && vite build

vite v5.2.7 building SSR bundle for production...
transforming...
"GraphQLDirective", "GraphQLInterfaceType" and "GraphQLObjectType" are imported from external module "graphql" but never used in "src/lib/model.ts", "src/lib/components/common/text/TypeLink.svelte", "src/lib/components/tags/TypeTag.svelte", "src/lib/components/tags/DirectiveTag.svelte", "src/lib/components/tags/NullableTag.svelte" and "src/routes/types/[type]/+page.svelte".
✓ 2521 modules transformed.
rendering chunks...
vite v5.2.7 building for production...
transforming...
✓ 2688 modules transformed.
rendering chunks...
computing gzip size...
.svelte-kit/output/client/_app/version.json                                      0.03 kB │ gzip:  0.05 kB
.svelte-kit/output/client/.vite/manifest.json                                   10.14 kB │ gzip:  1.33 kB
.svelte-kit/output/client/_app/immutable/assets/pages.BZ7dBxQt.css               0.05 kB │ gzip:  0.07 kB
.svelte-kit/output/client/_app/immutable/assets/ArgsList.tGPHACmx.css            0.08 kB │ gzip:  0.09 kB
.svelte-kit/output/client/_app/immutable/assets/DirectiveTag.DuCFScKg.css        0.08 kB │ gzip:  0.09 kB
.svelte-kit/output/client/_app/immutable/assets/1.DDNgGNlk.css                   0.13 kB │ gzip:  0.13 kB
.svelte-kit/output/client/_app/immutable/assets/FieldDetails.DpqSh-Zn.css        0.18 kB │ gzip:  0.14 kB
.svelte-kit/output/client/_app/immutable/assets/8.lLT_wvMO.css                   0.29 kB │ gzip:  0.21 kB
.svelte-kit/output/client/_app/immutable/assets/ChevronDown.DFnX4Nf0.css         2.06 kB │ gzip:  0.67 kB
.svelte-kit/output/client/_app/immutable/assets/PreviousNextPage.CSTXT5K2.css    2.58 kB │ gzip:  0.81 kB
.svelte-kit/output/client/_app/immutable/assets/0.Dc85tgr_.css                 755.43 kB │ gzip: 66.56 kB
.svelte-kit/output/client/_app/immutable/entry/start.CVJiGpcK.js                 0.07 kB │ gzip:  0.08 kB
.svelte-kit/output/client/_app/immutable/chunks/index.CHsVb4y-.js                0.17 kB │ gzip:  0.14 kB
.svelte-kit/output/client/_app/immutable/chunks/stores.D-4leKqp.js               0.23 kB │ gzip:  0.17 kB
.svelte-kit/output/client/_app/immutable/nodes/2.BdoLL8Pg.js                     0.28 kB │ gzip:  0.23 kB
.svelte-kit/output/client/_app/immutable/chunks/control.DxVE8_x7.js              0.50 kB │ gzip:  0.29 kB
.svelte-kit/output/client/_app/immutable/chunks/index.Cx9QXIn7.js                0.93 kB │ gzip:  0.58 kB
.svelte-kit/output/client/_app/immutable/nodes/1.CShDa-Kn.js                     1.19 kB │ gzip:  0.73 kB
.svelte-kit/output/client/_app/immutable/nodes/6.BDp-HkmV.js                     1.54 kB │ gzip:  0.85 kB
.svelte-kit/output/client/_app/immutable/nodes/5.Cw50UrH9.js                     1.56 kB │ gzip:  0.86 kB
.svelte-kit/output/client/_app/immutable/nodes/7.DnI9rz4-.js                     1.58 kB │ gzip:  0.86 kB
.svelte-kit/output/client/_app/immutable/nodes/3.BNeAo0H9.js                     1.61 kB │ gzip:  0.87 kB
.svelte-kit/output/client/_app/immutable/chunks/DirectiveTag.DLldSsIg.js         2.66 kB │ gzip:  1.35 kB
.svelte-kit/output/client/_app/immutable/chunks/prism-json.CMKj7kFV.js           2.98 kB │ gzip:  1.28 kB
.svelte-kit/output/client/_app/immutable/nodes/4.BHlWSj7x.js                     3.44 kB │ gzip:  1.55 kB
.svelte-kit/output/client/_app/immutable/chunks/index.CMSEJhCx.js                4.71 kB │ gzip:  2.41 kB
.svelte-kit/output/client/_app/immutable/entry/app.C5ZPHdzr.js                   7.91 kB │ gzip:  3.07 kB
.svelte-kit/output/client/_app/immutable/chunks/scheduler.Eaj42bxH.js            9.70 kB │ gzip:  3.77 kB
.svelte-kit/output/client/_app/immutable/chunks/Button.BsVMG643.js              12.49 kB │ gzip:  3.90 kB
.svelte-kit/output/client/_app/immutable/chunks/ArgsList.BYUT_pzB.js            14.47 kB │ gzip:  4.63 kB
.svelte-kit/output/client/_app/immutable/chunks/entry.DwgrG_M6.js               26.60 kB │ gzip: 10.52 kB
.svelte-kit/output/client/_app/immutable/chunks/ChevronDown.Bxlyoj4L.js         34.56 kB │ gzip: 12.19 kB
.svelte-kit/output/client/_app/immutable/nodes/8.ChgsFVhQ.js                    49.09 kB │ gzip: 10.35 kB
.svelte-kit/output/client/_app/immutable/chunks/PreviousNextPage.CVmfswbI.js    85.43 kB │ gzip: 18.38 kB
.svelte-kit/output/client/_app/immutable/chunks/FieldDetails.CSpJlwZO.js       155.76 kB │ gzip: 47.76 kB
.svelte-kit/output/client/_app/immutable/nodes/0.BXXCkpHn.js                   190.24 kB │ gzip: 53.70 kB
.svelte-kit/output/client/_app/immutable/chunks/pages.DbavW4Kl.js              273.41 kB │ gzip: 81.89 kB
✓ built in 11.09s
  301 /docs/ -> /docs/welcome

node:internal/event_target:1096
  process.nextTick(() => { throw err; });
                           ^
Error: 404 /types/Float does not begin with `base`, which is configured in `paths.base` and can be imported from `$app/paths` - see https://kit.svelte.dev/docs/configuration#paths for more info (linked from /docs/queries/event)
To suppress or handle this error, implement `handleHttpError` in https://kit.svelte.dev/docs/configuration#prerender
    at file:///private/var/folders/0w/4w2_lc2j6rg0j39hd2sycd9w0000gp/T/carbon-multi-page@4.2.0/node_modules/.pnpm/@sveltejs+kit@2.5.5_@sveltejs+vite-plugin-svelte@3.0.2_svelte@4.2.12_vite@5.2.7/node_modules/@sveltejs/kit/src/core/config/options.js:202:13
    at file:///private/var/folders/0w/4w2_lc2j6rg0j39hd2sycd9w0000gp/T/carbon-multi-page@4.2.0/node_modules/.pnpm/@sveltejs+kit@2.5.5_@sveltejs+vite-plugin-svelte@3.0.2_svelte@4.2.12_vite@5.2.7/node_modules/@sveltejs/kit/src/core/postbuild/prerender.js:65:25
    at visit (file:///private/var/folders/0w/4w2_lc2j6rg0j39hd2sycd9w0000gp/T/carbon-multi-page@4.2.0/node_modules/.pnpm/@sveltejs+kit@2.5.5_@sveltejs+vite-plugin-svelte@3.0.2_svelte@4.2.12_vite@5.2.7/node_modules/@sveltejs/kit/src/core/postbuild/prerender.js:195:4)
    at Object.fn (file:///private/var/folders/0w/4w2_lc2j6rg0j39hd2sycd9w0000gp/T/carbon-multi-page@4.2.0/node_modules/.pnpm/@sveltejs+kit@2.5.5_@sveltejs+vite-plugin-svelte@3.0.2_svelte@4.2.12_vite@5.2.7/node_modules/@sveltejs/kit/src/core/postbuild/prerender.js:184:22)
    at dequeue (file:///private/var/folders/0w/4w2_lc2j6rg0j39hd2sycd9w0000gp/T/carbon-multi-page@4.2.0/node_modules/.pnpm/@sveltejs+kit@2.5.5_@sveltejs+vite-plugin-svelte@3.0.2_svelte@4.2.12_vite@5.2.7/node_modules/@sveltejs/kit/src/core/postbuild/queue.js:40:42)
    at file:///private/var/folders/0w/4w2_lc2j6rg0j39hd2sycd9w0000gp/T/carbon-multi-page@4.2.0/node_modules/.pnpm/@sveltejs+kit@2.5.5_@sveltejs+vite-plugin-svelte@3.0.2_svelte@4.2.12_vite@5.2.7/node_modules/@sveltejs/kit/src/core/postbuild/queue.js:49:7

Node.js v21.7.1
 ELIFECYCLE  Command failed with exit code 1.


For a more detailed output, run with --stacktrace
```

### System Info

```shell
npx envinfo --system --binaries --browsers --npmPackages "{@magidoc/*}"

  System:
    OS: macOS 14.4.1
    CPU: (10) arm64 Apple M1 Pro
    Memory: 1.03 GB / 32.00 GB
    Shell: 3.2.57 - /bin/bash
  Binaries:
    Node: 21.7.1 - ~/.nvm/versions/node/v21.7.1/bin/node
    Yarn: 1.22.19 - ~/.yarn/bin/yarn
    npm: 10.5.0 - ~/.nvm/versions/node/v21.7.1/bin/npm
    pnpm: 8.15.5 - ~/Library/pnpm/pnpm
  Browsers:
    Chrome: 123.0.6312.87
    Safari: 17.4.1
```

### Severity

Blocking all usage of Magidoc

**Comment by pelletier197** [comment]
Thank you for raising this. I will have a look as soon as possible!

**Comment by pelletier197** [comment]
I have just released 4.2.1 with the fix. This is something linked to svelte-kit, I think they might have introduced a bug in the latest version, but I have not been able to reproduce unfortunately. Let me know if you face any other issue with this version :slightly_smiling_face:
